### PR TITLE
Added blog section cards

### PR DESCRIPTION
**Pull request by omeersari**

### CODE REVIEW

File: components/BlogCard/BlogCard.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import styles from "./BlogCard.module.scss";
+import Icon from "../Icon/Icon";
+import contentData from "./contentData.json";
+
+type Props = {
+  blog: IBlogCard;
+};
+
+const BlogCard = () => {
+  const cardArray: any = contentData;
```
Comment (OrhanOzkercin) [inline components/BlogCard/BlogCard.tsx:13]: Is there any reason we did not give a type to cardArray?

File: components/BlogCard/BlogCard.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              />
+            </div>
+            <div className={styles.header}>
+              <div className={styles.hr}></div>
+              <div className={styles.title}>
+                {blog.title}
+                <div className={styles.readMore}>
+                  <Link href={blog.path}>
+                    <span className="flex items-center gap-2">
+                      Read More <Icon icon="arrow-right" size={10} />{" "}
```
Comment (OrhanOzkercin) [inline components/BlogCard/BlogCard.tsx:35]: I think empty space is mistake at the end of the line

File: components/BriefSection/BriefSection.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ const BriefSection = () => {
   return (
     <div className="flex flex-col justify-center items-center py-12">
       <div className="flex flex-col items-center">
-        <Title level={2}>Frontendship'e neden katılmalıyım?</Title>
+        <Title level={2}>Frontendship&apos;e neden katılmalıyım?</Title>
```
Comment (OrhanOzkercin) [inline components/BriefSection/BriefSection.tsx:12]: Instead of removing `'` from the text we should fix the problem about `'`

**Review by OrhanOzkercin — CHANGES_REQUESTED** [review]
Well done 👏 I think we should update `any` types. Other than that good job 💐

File: components/BlogCard/BlogCard.tsx (hunk 2 above)
Comment (omeersari) [reply components/BlogCard/BlogCard.tsx:35]: yes, that is correct let me fix it.

File: components/BlogCard/BlogCard.tsx (hunk 1 above)
Comment (omeersari) [reply components/BlogCard/BlogCard.tsx:13]: it will be fixed later. It is like this to visualise cards for now.

**Pull request merged**